kustomize/components/clairpgupgrade/clair-pg-upgrade.job.yaml

```diff
@@ -95,4 +95,4 @@ spec:
             - "run-postgresql"
           args:
             - "--version"
-  backoffLimit: 20
+  backoffLimit: 50
```

Comment:
If the problem is that kubelet is killing postgres without giving it a chance to delete its pid file, increasing backoff limit won't help — the job doesn't try to recover from an ungraceful postgres shutdown.